src/Fable.ReactGoogleMaps/README.md

```diff
@@ -134,3 +134,33 @@ let myMap =
 let bounds = mapRef.GetBounds()
 
 ```
+
+### Set bounds of map to fit all the markers
+
+```fs
+    // ...
+    let markerPositions: Position list = // ...
+        
+    // In some scenarios ref can be null, f.ex. slow network when the google maps isn't fully loaded yet.
+    let setRef (ref:obj) =
+        let bounds = ReactGoogleMaps.Coordinates.newLatLngBounds()
+        
+        match Option.ofObj ref with
+        | Some ref ->
+            markerPositions
+            |> List.fold (fun (acc:LatLngBounds) pos ->
+                acc.extend(!^ pos)
```

Comment:
what'S this doing?

Reply:
Adds all `LatLngLiteral` to the newly created bounds. [extend](https://developers.google.com/maps/documentation/javascript/reference/coordinates#LatLngBounds.extend) returns `LatLngBounds`.

If you have say four markers on your map, you can add the locations to the bounds in order to know how to have the correct zoom and location so that your maps shows all four markers.

This is handy if you don't really know anything of your marker locations upfront.